backend/python/app/models/comment.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+from . import db
+
+
+class Comment(db.Model):
+
+    __tablename__ = "comments"
+
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    story_translation_content_id = db.Column(
+        db.Integer,
+        db.ForeignKey("story_translation_contents.id"),
+        index=True,
+        nullable=False,
+    )
+
+    user_id = db.Column(db.Integer, db.ForeignKey("users.id"), nullable=False)
+
+    comment_index = db.Column(db.Integer, nullable=False)
+    time = db.Column(db.Integer, nullable=False)
+    resolved = db.Column(db.Boolean, nullable=True)
```

Comment:
uhg sorry, im flip flopping again - "Unlike SQLAlchemy, all boolean columns are not nullable and False by default."
https://sqlalchemy-defaults.readthedocs.io/en/latest/#:~:text=All%20string%20columns%20are%20not,nullable%20and%20False%20by%20default.

I think we should actually do `nullable=False` lmfao